sdk/identity/identity/src/util/tracingUtils.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { TracerProxy, Span, SupportedPlugins, GetTokenOptions } from "@azure/core-http";
+
+/**
+ * Creates a span with the given name and spanOptions.
+ *
+ * @param name Name of the span
+ * @param spanOptions Span Options
+ */
+export function createSpan(name: string, options?: GetTokenOptions): Span {
+  const spanOptions = options && options.spanOptions;
+  const tracer = TracerProxy.getTracer();
+  const span = tracer.startSpan(name, spanOptions);
+  return span;
+}
+
+/**
+ * Modify the span options of the options to the new parent span.
+ *
+ * @param span Span to be assigned as parent span
+ * @param options options to be modified
+ */
+
+export function modifySpanOptions(span: Span, options?: GetTokenOptions): GetTokenOptions {
```

Comment:
Really this isn't modifying span options so much as it is creating a new span options for the child span manufactured in createSpan. 

Since this method always gets called after createSpan, would it make more sense to have createSpan return an object of { span, options } so you can destructure the result and not need to call two methods each time?

Reply:
Yes, please.  I was about to make a similar suggestion in another place in this PR.  I think it's inconvenient to always have to call two methods, and also possibly easy to forget.